script/deploy-canary-demo

```diff
@@ -2,6 +2,7 @@
 set -euxo pipefail
 #if($1) canarySuffix=$1 else canarySuffix=nothing
 canarySuffix=${1+$1}
+chimeSDKMediaPipelinesStackId="CSDKMP"
```

Comment:
[Confirming] If this is a bit more detailed, does it violate the length of the stack name? Want to know if can make this a bit more readable :)

Reply:
we have [128 chars](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-console-create-stack-parameters.html), so I've made it a bit more detailed